### PR TITLE
Uniform batch scripts

### PR DESCRIPTION
**Pull request by un-tone**
For https://github.com/StarcounterApps/SignIn/issues/73

### CODE REVIEW

File: package.bat [hunk 1]
```diff
@@ -0,0 +1,2 @@
+cd src\SignIn
```
Comment (joozek78) [inline package.bat]: please use pushd / popd and %~dp0
@warpech can you update the gitbook? It looks like omission by mistake

Comment (un-tone) [inline package.bat]: Fixed.

Comment (warpech) [inline package.bat]: I have updated Gitbook, thanks for the catch.

File: src/SignIn/SignIn.csproj [hunk 2]
```diff
@@ -20,7 +20,7 @@
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>..\..\bin\Debug\</OutputPath>
+    <OutputPath>bin\Debug\</OutputPath>
```
Comment (warpech) [inline src/SignIn/SignIn.csproj]: The "Release" config output path should be changed too.

Comment (un-tone) [inline src/SignIn/SignIn.csproj]: Fixed

**Comment by warpech** [comment]
@KmaTTi89 pls review and add the recent changes

**Pull request merged**